tools/licensify.py

```diff
@@ -16,83 +16,89 @@
 import tempfile
 
 licenseRows = [
-  'Copyright 2016 Attic Labs, Inc. All rights reserved.',
-  'Licensed under the Apache License, version 2.0:',
-  'http://www.apache.org/licenses/LICENSE-2.0',
+    'Copyright 2016 Attic Labs, Inc. All rights reserved.',
+    'Licensed under the Apache License, version 2.0:',
+    'http://www.apache.org/licenses/LICENSE-2.0',
 ]
 
 comment_markers = {
-  'go': ('', '// ', ''),
-  'js': ('', '// ', ''),
-  'py': ('', '# ', ''),
-  'html': ('<!--', '  ', '-->'),
-  'css': ('/**', ' * ', ' */'),
+    'go': ('', '// ', ''),
+    'js': ('', '// ', ''),
+    'py': ('', '# ', ''),
+    'html': ('<!--', '  ', '-->'),
+    'css': ('/**', ' * ', ' */'),
 }
 
 
 def main():
-  files = subprocess.check_output(['git', 'ls-files']).split('\n')
-  for n in files:
-    if n != '' and n.find('vendor/') == -1:
-      _, ext = os.path.splitext(n)
-      if ext == '':
-        continue
-      ext = ext[1:]
-      pattern = buildLicensePattern(ext)
-      if pattern != None:
-        with open(n, 'r+') as f:
-          processFile(f, ext, pattern)
-
-
-# Updates the license block in file |f|.
+    files = subprocess.check_output(['git', 'ls-files']).split('\n')
+    for n in files:
+        if n != '' and n.find('vendor/') == -1:
```

Comment:
I didn't change this but I agree. I might also add a check for `.min.js` since the script now adds a header to js/perf/viewer/modules/Chart.bundle.min.js